protocol/x/blocktime/keeper/keeper.go

```diff
@@ -114,17 +114,10 @@ func (k Keeper) UpdateAllDowntimeInfo(ctx sdk.Context) {
 
 // GetDowntimeInfoFor gets the DowntimeInfo for a specific duration. If the exact duration is not observed, it
 // returns the DowntimeInfo for the largest duration that is smaller than the input duration. If the input
-// duration is smaller than all observed durations, then return a DowntimeInfo with duration 0 and the current
-// block's info.
+// duration is smaller than all observed durations, then return an empty DowntimeInfo.
```

Comment:
So originally when we implemented this module, we wanted to always take the most pessimistic case. Since we don't track downtime for anything shorter than the shortest, we just always assume downtime has occurred. 

Any particular needd to change this here?